pkg/operator/operator.go

```diff
@@ -271,5 +274,27 @@ func (o *Operator) Config() *manifests.Config {
 		return manifests.NewDefaultConfig()
 	}
 
+	cmap, err = o.client.KubernetesInterface().CoreV1().ConfigMaps("kube-system").Get("cluster-config-v1", metav1.GetOptions{})
```

Comment:
I feel we should make aware that storing the pull secret in a configmap might not be the best place?!

Reply:
Agreed

Reply:
Absolutely, we should raise this in openshift/installer